geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/QueryStrategyDecider.scala

```diff
@@ -21,13 +20,12 @@ import org.opengis.filter.{And, Filter, Id, PropertyIsLike}
 import scala.collection.JavaConversions._
 
 trait VersionedQueryStrategyDecider {
-  def chooseStrategy(sft: SimpleFeatureType, query: Query, hints: StrategyHints, version: Int): Strategy
+  def chooseStrategy(sft: SimpleFeatureType, query: Query, hints: StrategyHints): Strategy
```

Comment:
Hmm... I do like changing the API here.

I'm wondering if we'll wind up in a situation where we modify the API and wind up with an incompatible collection of data and api versions.